e2e/environment.test.ts

```diff
@@ -0,0 +1,156 @@
+import { loginAsAdmin, navigateTo } from './test-util';
+import { expect, test } from '@playwright/test';
+
+test.describe('environment ', () => {
+  test('Rendering Image List', async ({ page }) => {
+    await loginAsAdmin(page);
+    await navigateTo(page, 'environment');
+    await page.waitForTimeout(1000);
+    const table = page.locator('.ant-table-content');
+    await expect(table).toBeVisible();
+  });
+
+  test('user can install image', async ({ page }) => {
+    await loginAsAdmin(page);
+    await navigateTo(page, 'environment');
+
+    // Sort installation status
+    await page.locator('.ant-table-cell.ant-table-column-sort').first().click();
+
+    // Find uninstalled item and select
+    const uninstalledImage = page
+      .locator('.ant-table-cell.ant-table-column-sort')
+      .filter({
+        hasNot: page.locator('.ant-tag-gold'),
+      })
+      .nth(1);
+    // If all images are installed, skip the test
+    const count = await uninstalledImage.count();
+    if (count === 0) {
+      test.skip();
+    }
+    await uninstalledImage.click();
+
+    // Install image
+    await page
+      .getByRole('button', { name: 'vertical-align-bottom Install' })
+      .click();
+    await page.getByRole('button', { name: 'Install', exact: true }).click();
+    await expect(
+      page.getByText('It takes time so have a cup of coffee!'),
+    ).toBeVisible();
+
+    // Verify installing status
+    const installingItem = page
+      .locator('.ant-table-row > td:nth-child(2)')
+      .first();
+    await expect(installingItem.getByText('installing')).toBeVisible();
+  });
+
+  test('user can modify image resource limit', async ({ page }) => {
+    await loginAsAdmin(page);
+    await navigateTo(page, 'environment');
+    // Click resource limit button
+    await page
+      .locator('.ant-table-row > td:nth-child(12)')
```

Comment:
Please use `findColumnIndex` function instead of `css selector`

Reply:
I modified it 
https://github.com/lablup/backend.ai-webui/pull/2665/commits/3467f2a0a3a7d6991acb40db33574fa021e7c060